components/profile-controller/controllers/profile_controller.go

```diff
@@ -325,109 +329,84 @@ func (r *ProfileReconciler) appendErrorConditionAndReturn(ctx context.Context, i
 func (r *ProfileReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&profilev1.Profile{}).
-		Owns(&istiorbac.ServiceRole{}).
-		Owns(&istiorbac.ServiceRoleBinding{}).
 		Owns(&corev1.Namespace{}).
 		Owns(&corev1.ServiceAccount{}).
 		Owns(&rbacv1.RoleBinding{}).
 		Complete(r)
 }
 
+func (r *ProfileReconciler) getSecurityCondition(profileIns *profilev1.Profile) []*istioSecurity.Condition {
+	if r.UserIdHeader == "" {
+		return []*istioSecurity.Condition{}
+	}
```

Comment:
will this condition mean that anybody will be able to access anything in case the userId header is not configured? I think this wasn't the case in the original Istio RBAC.